src/components/DocumentRenderer/LocalDocumentRendererContainer.tsx

```diff
@@ -25,7 +25,7 @@ export const LocalDocumentRendererContainer: FunctionComponent<NavigationProps>
     <View style={{ flex: 1 }}>
       <DocumentRenderer
         document={document.document}
-        goBack={navigation.goBack}
+        goBack={() => navigation.goBack()}
```

Comment:
What was the issue with the existing implementation?

Reply:
The existing one will pass down the navigation.goBack instantly, it may not have been initialised and as a result when child component is clicked, it doesnt navigate.